tasks/methodology/word-embeddings/README.md

```diff
@@ -1 +1,44 @@
-# Word Embeddings
+# Word Embeddings using Word2Vec.
+
+### Procedure
+
+1) I shall be working with [Fake News data](https://www.kaggle.com/mrisdal/fake-news)
+from Kaggle as an example for Word Embedding.
+This data set has sufficient data containing documents to train the model on.
+
+2) Clean/Tokenize the documents in the data set.
+
+3) Vectorize the model using Word2Vec and explore the results like finding most similar
+words and finding similarity.
+
+[gensim](https://radimrehurek.com/gensim/) package is used for Word2Vec functionality.
+
+### Running Locally
+
+python tasks/methodology/word-embeddings/word_embeddings.py <word1> <word2>
```

Comment:
- running python instead of mlflow is ok but it would not take care of dependency listed on conda.yaml so please use mlflow instead of python.
- instead of running `tasks/methodology/word-embeddings/word_embeddings.py word1 word2 topn` please update to first cd into `tasks/methodology/word-embeddings/word_embeddings.py` and then run as `mlflow run . -P word1=<word1> -P word2=<word2>`
- Please replace word1 and word2 with your favorite words or and an example. unfortunately my given words do not exist in the vocabulary and thrown error.

Reply:
I have resolved the comment and checked in the updated code.